lib/openstax/aws/stack_event.rb

```diff
@@ -0,0 +1,26 @@
+
+require_relative 'stack_status'
+
+module OpenStax::Aws
+  class StackEvent
```

Comment:
you can (and I think should) leave this as an inner class:

```ruby
module OpenStax::Aws
  class Stack
    class Event
      ...
    end
  end
end
```

We move it to `stack_event.rb` just because `stack.rb` is getting long and these inner classes are each long-ish.  But we still like the inner class scoping.